engines/python/setup/djl_python/tests/test_test_model.py

```diff
@@ -54,7 +55,8 @@ def test_with_env(self):
         envs = {
             "OPTION_MODEL_ID": "NousResearch/Nous-Hermes-Llama2-13b",
             "SERVING_LOAD_MODELS": "test::MPI=/opt/ml/model",
-            "OPTION_ROLLING_BATCH": "auto"
+            "OPTION_ROLLING_BATCH": "auto",
+            "TGI_COMPAT": "true"
```

Comment:
Can we use SERVING_TGI_COMPAT?

Reply:
Why SERVING?

Reply:
This should be a global server configuration and can be configure in `config.properties`